src/DssSpell.t.sol

```diff
@@ -634,4 +677,48 @@ contract DssSpellTest is DssSpellTestBase {
         assertEq(arbitrumGateway.validDomains(arbDstDomain), 0, "l2-arbitrum-invalid-dst-domain");
     }
 
+    function testDirectAaveV2Integration() public {
+        _vote(address(spell));
+        _scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        bytes32 ilk = "DIRECT-AAVEV2-DAI";
+        D3MHubLike hub = D3MHubLike(addr.addr("DIRECT_HUB"));
+        D3MAavePoolLike pool = D3MAavePoolLike(addr.addr("DIRECT_AAVEV2_DAI_POOL"));
+        D3MAavePlanLike plan = D3MAavePlanLike(addr.addr("DIRECT_AAVEV2_DAI_PLAN"));
+        D3MOracleLike oracle = D3MOracleLike(addr.addr("DIRECT_AAVEV2_DAI_ORACLE"));
+        D3MMomLike mom = D3MMomLike(addr.addr("DIRECT_MOM"));
+
+        // Do a bunch of sanity checks of the values that were set in the spell
+        (address _pool, address _plan, uint256 tau,,) = hub.ilks(ilk);
+        assertEq(_pool, address(pool));
+        assertEq(_plan, address(plan));
+        assertEq(tau, 7 days);
+        assertEq(hub.vow(), address(vow));
+        assertEq(hub.end(), address(end));
+        assertEq(mom.authority(), address(chief));
+        assertEq(pool.king(), pauseProxy);
```

Comment:
```suggestion
        assertEq(pool.hub(), address(hub));
        assertEq(pool.king(), pauseProxy);
```

Reply:
Maybe not necessary, as we could also check the other constructor values...

Reply:
FYI constructor values are checked in the init library which will execute in the spell.

Reply:
@gbalabasquer let me know if you still think this should be added

Reply:
Let's leave it as it is.